libs/lb/lb-rs/src/shared/account.rs

```diff
@@ -24,6 +29,97 @@ impl Account {
     pub fn public_key(&self) -> PublicKey {
         PublicKey::from_secret_key(&self.private_key)
     }
+
+    pub fn get_phrase(&self) -> SharedResult<[String; 24]> {
+        let key = self.private_key.serialize();
+        let key_bits = key.iter().fold(String::new(), |mut out, byte| {
+            let _ = write!(out, "{:08b}", byte);
```

Comment:
what does this do?

Reply:
`fold` lets you accumulate elements in an `Iterator`. Here I used it to transform the phrases into a byte string (and I do the same in my other use case). In my new commit, I removed this usage for a more concise method.

Reply:
I did this instead of collecting `format!(...)`s since Clippy gave me a warning (`fold` is more optimal).

Reply:
I mean `:08b`

Reply:
That formats a number into its 8 bit representation.